content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -89,8 +89,6 @@ A given Kubernetes server will only preserve a historical list of changes for a
 
 ### Watch bookmarks
 
-{{< feature-state for_k8s_version="v1.16" state="beta" >}}
-
 To mitigate the impact of short history window, we introduced a concept of `bookmark` watch event. It is a special kind of event to pass an information that all changes up to a given `resourceVersion` client is requesting has already been send. Object returned in that event is of the type requested by the request, but only `resourceVersion` field is set, e.g.:
```

Comment:
@wojtek-t , do you think it would be worthwhile to update the feature-state shortcode (version=v1.17, state=stable)?

Reply:
I'm not sure - I don't think we state the feature status for GA features.

Reply:
It's not mandatory, but when features go GA it's handy to signal that to readers. Stating stability from v1.17 also nudges people to upgrade if they want the feature.
(after adding this kind of labelling it eventually needs cleanup - which is a job for a bot).